hazelcast/src/main/java/com/hazelcast/internal/metrics/metricsets/OperatingSystemMetricSet.java

```diff
@@ -107,14 +111,18 @@ public double get(Object bean) throws Exception {
      *
      * @param source     the source object.
      * @param methodName the name of the method to retrieve.
+     * @param name the probe name
      * @return the method
      */
-    private static Method getMethod(Object source, String methodName) {
+    private static Method getMethod(Object source, String methodName, String name) {
         try {
-            Method method = source.getClass().getDeclaredMethod(methodName);
-            method.setAccessible(true);
+            Method method = source.getClass().getMethod(methodName);
```

Comment:
I'm very worried that we are now going to loose a bunch of metrics. And based on your tests, this seems to be the case. This is not acceptable.